### ISSUE TITLE
Invalid CSRF token on v3.0.1

### ISSUE THREAD

**Issue by MrSimonC** [olsh/todoist-net#20]
Hello. Happy Christmas (or happy holidays)!

Since Todoist.net have deprecated v8 of the API, moving to v9, I've updated from v2.2.2 to v3.0.1.

My code hasn't changed since both versions, but now, every time, I'm receiving `Invalid CSRF token`:

<img width="879" alt="Screenshot 2022-12-27 at 11 02 26" src="https://user-images.githubusercontent.com/3033044/209657721-5ca5e486-9bd0-458d-b58d-4358192f1f2d.png">

I've created a [test-repo](https://github.com/MrSimonC/Testing-broken-todoist) with the simplest code to show the issue:

```csharp
using Todoist.Net;

var todoistKey = Environment.GetEnvironmentVariable("TODOIST_APIKEY");
var client = new TodoistClient(todoistKey);
var result = await client.Projects.GetAsync(); // errors here: 409 Invalid CSRF token
```

I hit this error running in the cloud: GitHub code-spaces, Azure functions and Azure Container apps. It's definitely code related.
Can you / someone please pull the repo and check if you get the same thing?

**Comment by MrSimonC** [comment]
I've also just ran a hello-world starter test just now from the Getting Started guide on todoist.net and can confirm the `curl` commands work fine with the same API key I'm using on both sync and rest API endpoints...

**Comment by olsh** [comment]
Hello @MrSimonC 

Merry Christmas! 🎅🏻 

Yes, they changed the authorization flow recently (v9 worked a couple of days ago).
I will roll out a fix soon. Thank you 👍🏻

**Comment by olsh** [comment]
The issue is fixed in [v4.0.0](https://www.nuget.org/packages/Todoist.Net/4.0.0)
Thank you! 👍🏻
